components/public-api/gitpod/experimental/v1/user.proto

```diff
@@ -0,0 +1,96 @@
+syntax = "proto3";
+
+package gitpod.experimental.v1;
+
+option go_package = "github.com/gitpod-io/gitpod/public-api/experimental/v1";
+
+import "google/protobuf/timestamp.proto";
+
+message User {
+    // id is a UUID of the user
+    string id = 1;
+
+    string name = 2;
+
+    string avatar_url = 3;
+
+    // created_at is the creation time
+    google.protobuf.Timestamp created_at = 5;
+}
+
+message PublicSSHKey {
+    // id is a UUID of the SSH key
+    string id = 1;
+
+    // name is the name of the SSH key
+    string name = 2;
+
+    // key is the public SSH key
+    string key = 3;
+
+    // created_at is the creation time
+    google.protobuf.Timestamp created_at = 4;
+}
+
+service UserService {
+    // Lists the public SSH keys.
+    rpc GetUser(GetUserRequest) returns (GetUserResponse) {};
+
+
+    // Lists the public SSH keys.
+    rpc ListPublicSSHKeys(ListPublicSSHKeysRequest) returns (ListPublicSSHKeysResponse) {};
+
+    // Adds a public SSH key.
+    rpc CreatePublicSSHKey(CreatePublicSSHKeyRequest) returns (CreatePublicSSHKeyResponse) {};
+
+    // View extended details for a single public SSH key.
+    rpc GetPublicSSHKey(GetPublicSSHKeyRequest) returns (GetPublicSSHKeyResponse) {}
+
+    // Removes a public SSH key.
+    rpc DeletePublicSSHKey(DeletePublicSSHKeyRequest) returns (DeletePublicSSHKeyResponse) {}
+}
+
+message GetUserRequest {
```

Comment:
If this takes no arguments, how do you identify the user you're getting?

Reply:
If this is intended to get details of the current credentials, then we should rename.

Normally, `GetUser` should take an ID of the `user` you're retrieving. This is because it's not only the currently logged in user that may be retrieving details of the user - it may be SSO or other system which creates the users.